torch/csrc/cudnn/Conv.cpp

```diff
@@ -163,53 +203,96 @@ struct algorithm_search<cudnnConvolutionFwdAlgo_t> {
          CUDNN_CONVOLUTION_FWD_ALGO_WINOGRAD,
          CUDNN_CONVOLUTION_FWD_ALGO_WINOGRAD_NONFUSED,
     };
-    size_t max_ws_size = getMaxWorkspaceSize<cudnnConvolutionFwdAlgo_t>(handle,conv,algo,sizeof(algo)/sizeof(algo[0]),state);
+    size_t max_ws_size = getMaxWorkspaceSize<cudnnConvolutionFwdAlgo_t>(
+        handle, conv, algo, sizeof(algo)/sizeof(algo[0]), state);
     Workspace ws(state, max_ws_size);
-
-    CHECK(cudnnFindConvolutionForwardAlgorithmEx(handle, conv.idesc.desc, in,
-        conv.wdesc.desc, wght, conv.cdesc.desc, conv.odesc.desc, out, 1, &algoCount,
-        &perfResults, ws.data, ws.size));
-
+    CHECK(cudnnFindConvolutionForwardAlgorithmEx(
+        handle,
+        conv.idesc.desc,
+        in,
+        conv.wdesc.desc,
+        wght,
+        conv.cdesc.desc,
+        conv.odesc.desc,
+        out,
+        1,
+        &algoCount,
+        &perfResults,
+        ws.data,
+        ws.size));
```

Comment:
I liked it more before 😞

Reply:
it was hard for me to parse the code so I made these changes